docs/14-ui.mdx

```diff
@@ -56,6 +56,8 @@ A [screen element](#screen-elements) is useful because it's an actor, it is rend
 
 The trick is _positioning the game UI above Excalibur_ and reacting to Excalibur events appropriately. With some tweaks to the way you embed Excalibur, this can be accomplished without too much code.
 
+Do not forget to set [PointerScope.Canvas](https://excaliburjs.com/docs/api/edge/enums/_input_pointer_.pointerscope.html#canvas), otherwise Excalibur will capture all mouse input and HTML GUI won't work properly.
```

Comment:
This can actually be shortened to `[[PointerScope.Canvas]]`

Reply:
Sure thing! Done.